app/jobs/submit_patron_request_job.rb

```diff
@@ -36,6 +36,7 @@ def perform(patron_request)
   #     The item has an existing request queue (TODO: https://github.com/sul-dlss/sul-requests/issues/2234)
   def send_to_illiad?(patron_request, item)
     return true if patron_request.scan?
+    return true if item.status == Folio::Item::STATUS_AGED_TO_LOST
     return false unless patron_request.patron.ilb_eligible?
```

Comment:
```suggestion
    return false unless patron_request.patron.ilb_eligible?
    return true if item.status == Folio::Item::STATUS_AGED_TO_LOST
```

maybe?

@saseestone: what should we do with aged-to-lost items for patrons that aren't eligible for ILL?

Reply:
Confirmed at standup that patrons who aren't otherwise eligible shouldn't get sent to ILLiad.